Formula/l/logdy.rb

```diff
@@ -0,0 +1,19 @@
+class Logdy < Formula
+  desc "Web based real-time log viewer"
+  homepage "https://logdy.dev"
+  # uncomment with the next release
+  # url "https://github.com/logdyhq/logdy-core/archive/refs/tags/v0.9.0.tar.gz"
+  url "https://github.com/logdyhq/logdy-core/releases/download/v0.9.0/logdy-core.tar.gz"
+  sha256 "bd55b3cd380117288112403a19f4e7deeffc667df8970425c5e1ce8b87848720"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-X 'main.Version=0.9.0'", "-o", bin/"logdy"
+  end
+
+  test do
+    assert_match "logdy version", shell_output("#{bin}/logdy --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
that seems a bit complicated, you can just do a server start and assert the log? or even if it is console log, you should be able to assert it as well.